tests/test_authentication.py

```diff
@@ -0,0 +1,467 @@
+from base64 import b64encode
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.test import APIClient
+from rest_framework import serializers
+from django.core import mail
+from django.urls import reverse
+
+from authors.apps.authentication.models import User, UserManager
+from authors.apps.authentication.serializers import (
+    LoginSerializer, RegistrationSerializer, UserSerializer)
+
+
+
+class UserTestCase(TestCase):
```

Comment:
Kindly seek @richien's help on having the different test cases in their own files.
In this case they can go into files that already exist (tests for serializers can be in the same files as views if they are both concerning authentication or something similar).

Reply:
@RonKbS  I have redistributed to files that test different features kindly please take a look